Clarify/add instructions in README.md (#139)
tweetback/tweetback: README.md

* Clone/download this repository
* In your terminal, `cd` to the folder of the project

> * Install [Node.js](https://nodejs.org/) (there are many ways to do this depending on your system and preferences, but if you are on a Mac and have [homebrew](https://brew.sh) installed, an easy installation method on is to run `brew install node`)

Reviewer: ```suggestion
* Install [Node.js](https://nodejs.org/) (there are many ways to do this depending on your system and preferences; most package managers provide a package for it)
```

* Run `npm install`

### Populate the database from your Twitter Archive zip

1. Copy `./data/tweets.js` (or `./data/tweet.js`) from your [Twitter Archive](https://help.twitter.com/en/managing-your-account/how-to-download-your-twitter-archive) `zip` file into the `./database` directory of this project.

>    * Rename `window.YTD.tweets.part0` (or `window.YTD.tweet.part0`) in `tweets.js` to `module.exports`

Reviewer: Since many of the strings in here only differ only by a single letter, I wonder if it might be better just to add a single comment at the top like, "Note: 'tweet' may or may not be pluralized as 'tweets' throughout" somewhere at the top? Instead of noting each individual potential variation, that is.

Reviewer: See also discussion in #65 
(edit: looks like it was already mentioned; never mind)

   * Rename `window.YTD.tweets.part0` or (`window.YTD.tweet.part0`) in `twitter-circle-tweet.js` to `module.exports`
1. Run `npm run import` or `npm run import-without-circles`

### Build the web site

1. Edit the `_data/metadata.js` file to add metadata information.
   * `username` is your Twitter username (no @ symbol)
   * `homeLabel` and `homeUrl` are the the name and URL of a homepage or other link to include at the top of the main archive page
   * `data.avatar` should link to a 16x16 pixel profile picture; the default will use the favicon from the website at `homeUrl`
1. Run `npm run build` (will just create the proper files) or `npm start` (will run a server to look at them in your browser).
	* _Optional:_ If you want the web site to live in a subdirectory (e.g. `/twitter/`), use [Eleventy’s Path Prefix feature](https://www.11ty.dev/docs/config/#deploy-to-a-subdirectory-with-a-path-prefix) via the command line `--pathprefix=twitter` or via a return object in your configuration file. Careful: this is an option to Eleventy and not npm, so it needs to live after a `--` separator (for instance, `npm run build -- --pathprefix=twitter`).

> ⚠️ _Warning_: the first build may take quite a long time (depending on the size of your archive), as remote media is fetched/downloaded into your project locally. There is currently no progress bar so be patient! Repeat builds will be much faster.

Reviewer: I'd cross-reference this against issue #43
